.github/workflows/Container-Ubuntu-22.yml

```
@@ -0,0 +1,28 @@
+# GitHub Action Workflow for building the Ubuntu 22 images.
+#
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+
+name: "Ubuntu 22 Containers"
+
+# This workflow only runs (on the main branch or on PRs targeted
+# at the main branch) and if files inside the Ubuntu-22 directory
+# have been modifed/added/removed...
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main ]

```

If you're going to use a multi-line array for `paths`, it'd be consistent to do the same for `branches`.

```suggestion
    branches:
      - main

```